torch/nn/init.py

```diff
@@ -235,19 +242,21 @@ def xavier_uniform_(tensor, gain=1.):
     Args:
         tensor: an n-dimensional `torch.Tensor`
         gain: an optional scaling factor
+        is_depthwise: an optional indicator indicate whether a depthwise conv kernel,
```

Comment:
just some phrasing: "is_depthwise: wether the considered weights belong to a depthwise convolution (with `group > 1`)".